### PR TITLE
Template: Fix GitHub Actions CI and Linting badges links

### PR DESCRIPTION
**Pull request by RaqManzano**
Fixed GitHub Actions CI and Linting badges links.

### CODE REVIEW

File: nf_core/pipeline-template/README.md [hunk 1]
```diff
@@ -9,8 +9,8 @@
 
 {% endif -%}
 {% if github_badges -%}
-[![GitHub Actions CI Status](https://github.com/{{ name }}/workflows/nf-core%20CI/badge.svg)](https://github.com/{{ name }}/actions?query=workflow%3A%22nf-core+CI%22)
-[![GitHub Actions Linting Status](https://github.com/{{ name }}/workflows/nf-core%20linting/badge.svg)](https://github.com/{{ name }}/actions?query=workflow%3A%22nf-core+linting%22){% endif -%}
+[![GitHub Actions CI Status](https://github.com/{{ name }}/actions/workflows/ci.yml/badge.svg)](https://github.com/{{ name }}/actions?query=workflow%3A%22nf-core+CI%22)
```
Comment (ewels) [inline nf_core/pipeline-template/README.md:12]:
```suggestion
[![GitHub Actions CI Status](https://github.com/{{ name }}/actions/workflows/ci.yml/badge.svg)](https://github.com/{{ name }}/actions/workflows/ci.yml)
```

File: nf_core/pipeline-template/README.md [hunk 2]
```diff
@@ -9,8 +9,8 @@
...
+[![GitHub Actions Linting Status](https://github.com/{{ name }}/actions/workflows/linting_comment.yml/badge.svg)](https://github.com/{{ name }}/actions?query=workflow%3A%22nf-core+linting%22){% endif -%}
```
Comment (ewels) [inline nf_core/pipeline-template/README.md:13]: Linting, not linting comment :)

```suggestion
[![GitHub Actions Linting Status](https://github.com/{{ name }}/actions/workflows/linting.yml/badge.svg)](https://github.com/{{ name }}/actions/workflows/linting.yml){% endif -%}
```

**Review by ewels — APPROVED** [review]
Nice! One minor change - to update the link to the CI runs as well as just the badge.

Also you put the linting comment workflow rather than the linting :) The linting comment CI is the job that posts a comment after linting fails, which is kind of irrelevant. It always passes even if linting is failing.

**Comment by ewels** [comment]
@nf-core-bot changelog

**Review by mashehu — APPROVED** [review]

File: nf_core/pipeline-template/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ewels) [reply nf_core/pipeline-template/README.md]: Is the `%20` needed? That's a URL-encoded `.`?

[the remark this answers is not in the document]
Comment (mashehu) [reply nf_core/pipeline-template/README.md]: nah, it's fine as it is.

**Review by maxulysse — APPROVED** [review]

**Pull request merged**